test_Lusine_Stepanyan.py

```diff
@@ -1,3 +1,2 @@
 def test_print_football_player_name(name="Leo Messi"):
```

Comment:
Add also surname and age properties and print them also

Reply:
done